.gitmodules

```diff
@@ -2,15 +2,10 @@
 	path = dumps/mongodb-sample-databases
 	url = https://github.com/mcampo2/mongodb-sample-databases.git
 	branch = master
-
 [submodule "dotnet-example"]
 	path = tests/dotnet-example
 	url = https://github.com/FerretDB/dotnet-example.git
 	branch = main
-[submodule "enmeshed-runtime"]
-	path = tests/enmeshed-runtime
-	url = https://github.com/nmshd/cns-runtime.git
-	branch = main
```

Comment:
We need that line; otherwise, dependabot will not update that submodule.

There is also no reason to change the order and group of submodules. Just edit one line.

Reply:
That was a result of doing `git rm` and then `git add submodule` etc., because I wasn't entirely sure if the submodule's code itself had changed.

Fixed.

Reply:
Well, the grouping change wasn't reverted. `dumps/` should be a separate group from `tests/` – as it was